internal/xpkg/config.go

```diff
@@ -13,13 +13,16 @@ import (
 
 const (
 	errListImageConfigs = "cannot list ImageConfigs"
+	errFindBestMatch    = "cannot find best matching ImageConfig"
 )
 
 // ConfigStore is a store for image configuration.
 type ConfigStore interface {
 	// PullSecretFor returns the name of the selected image config and
 	// name of the pull secret for a given image.
 	PullSecretFor(ctx context.Context, image string) (imageConfig, pullSecret string, err error)
+	// ImageVerificationConfigFor returns the ImageConfig for a given image.
+	ImageVerificationConfigFor(ctx context.Context, image string) (imageConfig string, verificationConfig *v1beta1.ImageVerification, err error)
```

Comment:
```suggestion
	ImageVerificationConfigFor(ctx context.Context, image string) (imageConfig string, iv *v1beta1.ImageVerification, err error)
```

Nit: I don't think this needs such a long/descriptive name given the type is pretty self-descriptive.